.github/workflows/reusable-accessibility.yml

```diff
@@ -12,49 +12,43 @@ on:
 
 jobs:
   playwright-accessibility-test:
-    defaults:
-      run:
-        working-directory: ${{ inputs.path }}
     runs-on: ubuntu-latest
     env:
       CI: true
     steps:
-      - name: Install chrome
-        uses: browser-actions/setup-chrome@c785b87e244131f27c9f19c1a33e2ead956ab7ce # v1
       - name: Get source code
         uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
       - uses: actions/setup-node@a0853c24544627f65ddf259abe73b1d18a591444 # v5
         with:
           node-version: ${{ inputs.node-version }}
           cache: 'npm'
-          cache-dependency-path: ${{ inputs.path }}/package-lock.json
-
-      - name: CI Common
-        working-directory: common
-        run: |
-          npm ci
+          cache-dependency-path: package-lock.json
 
       - name: Install Node dependencies
         run: |
           npm ci
-          npx playwright install chromium msedge
+          npm install serve -g
```

Comment:
I'm confused by this change. Before we were installing the browsers needed for Playwright tests, and now we're installing a package globally? How are these related?